Week 1/Weekly Practice Problems (Week 1)/2011: Final Value of Variable After Performing Operations/dominic.cpp

```diff
@@ -0,0 +1,10 @@
+class Solution {
+public:
+    int finalValueAfterOperations(vector<string>& operations) {
+        int x = 0;
+        for (string op : operations) {
```

Comment:
```suggestion
        for (string& op : operations) {
```
use reference to avoid copying